src/tpl/feature.html

```diff
@@ -0,0 +1,116 @@
+{{#*inline "browserStatus"}}
```

Comment:
Can we somehow share the code between the two templates? Or are they totally different?